abci/client/grpc_client.go

```diff
@@ -88,8 +89,7 @@ func (cli *grpcClient) OnStart() error {
 
 RETRY_LOOP:
 	for {
-		//nolint:staticcheck // SA1019 Existing use of deprecated but supported dial option.
-		conn, err := grpc.Dial(cli.addr, grpc.WithInsecure(), grpc.WithContextDialer(dialerFunc))
+		conn, err := grpc.Dial(cli.addr, grpc.WithTransportCredentials(insecure.NewCredentials()), grpc.WithContextDialer(dialerFunc))
```

Comment:
I do not know why the linter started complaining for this here, but this seemed like the best solution

Reply:
seems resolved by https://github.com/celestiaorg/celestia-core/pull/1006/commits/310cd2a83a6075804f64ca94f90068684aeb0c9c